samples/README.md

```diff
@@ -0,0 +1,48 @@
+# Bring Your Own Container Sample
+
+This sample demonstrates how to start training jobs using your own training script, packaged in a SageMaker-compatible container, using the Amazon AWS Controllers for Kubernetes (ACK) service controller for Amazon SageMaker.                     
+
+## Prerequisites    
+
+This sample assumes that you have already configured an EKS cluster with the ACK operator. It also assumes that you have installed `kubectl` - you can find a link on our [installation page](TODO).
+
+In order to follow this script, you must first create a training script packaged in a Dockerfile that is [compatible with Amazon SageMaker](https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html). The [Distributed Mask R-CNN sample](https://github.com/awslabs/amazon-sagemaker-examples/tree/master/advanced_functionality/distributed_tensorflow_mask_rcnn), published by the SageMaker team, contains a predefined training script and helper bash scripts for reference.
```

Comment:
A cool trick I've learnt from others is instead of inlining the URLs in links, you can replace it with a placeholder and then later in the document tell it what that placeholder should represent. For example:

```
... Docker that is [compatible with Amazon SageMaker][sagemaker-containers]. The ...

...

[sagemaker-containers]: https://docs.aws.amazon.com/sagemaker/latest/dg/amazon-sagemaker-containers.html
```

Feel free to use it in some places to help improve the flow of the text so it's easier to read in MD. But it's just syntactic sugar, so no big deal either way.